jmix-flowui/flowui/src/main/java/io/jmix/flowui/component/UiComponentUtils.java

```diff
@@ -131,6 +130,9 @@ public static Collection<Component> getOwnComponents(Component container) {
             return ((ComponentContainer) container).getOwnComponents();
         } else if (container instanceof HasComponents) {
             return container.getChildren().sequential().collect(Collectors.toList());
+        } else if (container instanceof View<?>) {
```

Comment:
Because view does not implement HasComponents or ComponentContainer, but it is composite component, and if your invoke this method for view, it throws error, and i use this API to fetch child components